addon/application/settings.js

```diff
@@ -2,5 +2,5 @@ import EmberObject from '@ember/object';
 import Evented from '@ember/object/evented';
 
 export default EmberObject.extend(Evented, {
-  localEnvironment: 'dev',
+  localEnvironment: 'prod',
```

Comment:
I wouldn't recommend having this default to `prod`

Reply:
I understand why defaulting to prod could have side effects, but I'm not sure of another way around this issue. The reason the DEV label flashes briefly is because this `localEnvironment` is used while the app's application settings are loading. The only other way I can think of to get rid of the flash would be to remove the default `localEnvironment` entirely. Thoughts?

Reply:
I think that might be okay.